src/qtism/data/content/BodyElement.php

```diff
@@ -615,4 +621,36 @@ public function hasAriaLabel()
     {
         return $this->ariaLabel !== '';
     }
+
+    /**
+     * @param bool $ariaHidden
+     * @throws InvalidArgumentException
+     */
+    public function setAriaHidden($ariaHidden)
+    {
+        if (!is_bool($ariaHidden)) {
+            $val = (is_object($ariaHidden)) ? ('instance of ' . get_class($ariaHidden)) : $ariaHidden;
```

Comment:
Unnecessary parentheses.

Reply:
Not so sure, operator precedence of `.` will IMHO make the thing inconsistent.

Reply:
Well it's not a big deal really. The code is doing what it must do. I approved already.

Just for the records:
The parentheses around the condition is not necessary at all, because it has only one member.
The ternary operator structure is higher level than the expression it contains, of course: 
```
echo true ? 'a' . 'b' : 'c' . 'd', "\n";      // => 'ab'
echo false ? 'a' . 'b' : 'c' . 'd', "\n";     // => 'cd'
```